pkg/endpoint/policy.go

```diff
@@ -439,19 +297,15 @@ func (e *Endpoint) updateNetworkPolicy(owner Owner) error {
 	}
 	deniedIngressIdentities := make(map[identityPkg.NumericIdentity]bool)
 	for srcID, srcLabels := range *e.LabelsMap {
-		if c.IngressIdentities[srcID] {
-			// Already allowed for L3-only.
```

Comment:
I guess this is a form of caching that we were doing before, which we no longer do.

Reply:
This is correct. Eventually I want to have a cilium.NetworkPolicy object contain the per-endpoint policy instead of having intermediate representations inside of the endpoint structure. This policy could then be plumbed down into a datapath package / interface which takes an endpoint and realizes its desired cilium.NetworkPolicy.